cfgov/paying_for_college/models/disclosures.py

```diff
@@ -187,6 +191,14 @@ def __str__(self):
         )
 
 
+def format_for_null(value):
```

Comment:
It's not standardized nicely, but we are trying hard not to affect the original tool, as it is still in heavy use and will stop students from enrolling if it breaks.

The format_for_null fixed a problem we had with the original API, which was errantly delivering the text string of "None" instead of the json value of null in some cases.